maintenance/enabling-bpf.md

```diff
@@ -54,18 +54,26 @@ eBPF mode has the following pre-requisites:
     
   If {{site.prodname}} does not detect a compatible kernel, {{site.prodname}} will emit a warning and fall back to standard linux networking.
   
-- On each node, the BPF filesystem must be mounted at `/sys/fs/bpf`.  This is required so that the BPF filesystem persists when {{site.prodname}} is restarted.  (If the filesystem does not persist then pods will temporarily lose connectivity when {{site.prodname}} is restarted.)
+- On each node, the BPF filesystem must be mounted at `/sys/fs/bpf`.  This is required so that the BPF filesystem persists 
+  when {{site.prodname}} is restarted.  (If the filesystem does not persist then pods will temporarily lose connectivity when
+  {{site.prodname}} is restarted.)
 - For best pod-to-pod performance, an underlying network that doesn't require Calico to use an overlay.  For example:
  
   - A cluster within a single AWS subnet.
   - A cluster using a compatible cloud provider's CNI (such as the AWS VPC CNI plugin).
   - An on-prem cluster with BGP peering configured.
   
-- The underlying network must be configured to allow VXLAN packets between {{site.prodname}} hosts.  In eBPF mode, VXLAN is used to forward traffic to Kubernetes NodePorts, while preserving source IP.
-- A stable way to address the Kubernetes API server. Since eBPF mode takes over from kube-proxy, {{site.prodname}} needs a way to reach the API server directly.
+  If you must use an overlay, we recommend that you use VXLAN, not IPIP.  This is because IPIP performs poorly with eBPF mode due 
+  to technical limitations in the kernel.
+  
+- The underlying network must be configured to allow VXLAN packets between {{site.prodname}} hosts.  In eBPF mode, 
+  VXLAN is used to forward traffic to Kubernetes NodePorts, while preserving source IP.
+- A stable way to address the Kubernetes API server. Since eBPF mode takes over from kube-proxy, {{site.prodname}} 
+  Creating an needs a way to reach the API server directly.
 - The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
 
-> **Note**: The default kernel used by EKS is not compatible with eBPF mode.  If you wish to try eBPF mode with EKS, follow [these instructions](./ebpf-and-eks).
+> **Note**: The default kernel used by EKS is not compatible with eBPF mode.  If you wish to try eBPF mode with EKS, 
+> follow [these instructions](./ebpf-and-eks), which explain how to set up a suitable cluster.
```

Comment:
This results in a blue hyperlink in a blue notes box, which makes it kinda hard to spot that its a link.  Not sure if there's a sane way around that however?